WWTMVC5/GetTourFile.aspx

```diff
@@ -1,46 +1,6 @@
-<%@ Page Language="C#" ContentType="image/png" %>
-<%@ Import Namespace="System.IO" %>
-<%@ Import Namespace="System.Net" %>
-<%@ Import Namespace="WWTWebservices" %>
-<%
-    string returnString = "Erorr: No URL Specified";
-    string url = "";
-    
-    string path = Server.MapPath(@"TourCache");
-
-    try
-    {
-        if (Request.Params["targeturl"] != null && Request.Params["filename"] != null)
-        {
-            url = Request.Params["targeturl"];
-
-            string targetfile = Request.Params["filename"];
-            string filename = path + "\\" + Math.Abs(url.GetHashCode()) + ".wtt";
-
-            if (!File.Exists(filename))
-            {
-                if (url.ToLower().StartsWith("http"))
-                {
-                    using (WebClient wc = new WebClient())
-                    {
-                        byte[] data = wc.DownloadData(url);
+<%@ Page Language="C#" ContentType="image/png" %> 
 
-                        //Response.ContentType = wc.ResponseHeaders["Content-type"].ToString();
-                        int length = data.Length;
-
-                        File.WriteAllBytes(filename, data);
-                        //Response.OutputStream.Write(data, 0, length);
-                    }
-                }
-            }
-
-
-            FileCabinet.Extract(filename, targetfile, Response);
-        }
-    }
-    catch (System.Exception e)
-    {
-        returnString = e.Message;
-        Response.Write(returnString);
-    }
-%>
\ No newline at end of file
+<%@ Import Namespace="WWT.Providers" %>
+<%
+	RequestProvider.Get<GetTourFileProvider>().Run(this);
```

Comment:
This file was missed when converting aspx files to C# providers. The implementation in the aspx is identical to the already existing GetTourFileProvider